pkgs/development/libraries/json-glib/default.nix

```
@@ -6,19 +6,46 @@
 , ninja
 , pkg-config
 , gettext
-, withIntrospection ? stdenv.buildPlatform == stdenv.hostPlatform
 , gobject-introspection
-, fixDarwinDylibNames
 , gi-docgen
+, libxslt
+, fixDarwinDylibNames
 , gnome
+, buildPackages
+, pkgsBuildBuild
+, writeText
+, runCommand
+, makeBinaryWrapper
 }:
 
+let
+  gobject-introspection-wrapper = runCommand "make-wrappers" {
+    depsBuildBuild = [ pkgsBuildBuild.makeBinaryWrapper ];
+  } ''
+    makeWrapper ${buildPackages.gobject-introspection.dev}/bin/g-ir-scanner \
```

```suggestion
    makeWrapper ${lib.getDev buildPackages.gobject-introspection}/bin/g-ir-scanner \
```
Always nice to use the helper function in case dev output gets removed.